build.yaml

```diff
@@ -0,0 +1,575 @@
+# vim ft=yaml:fdm=indent
+# To modify an existing job, simply update the dictionary 'jobs_type'
+# commit, push
+# Then click 'deploy' on the jenkins job related to this repository
+# available on the jenkins homepage.
+
+project: 'ClusterHQ/flocker'
+git_url: 'https://github.com/ClusterHQ/flocker.git'
+
+# common_cli contains YAML aliases for common cli operations
+# used during the build process
+common_cli:
+  hashbang: &hashbang |
+    #!/bin/bash -l
+    set -x
+    set -e
+    os() {
+      # Determine OS platform
```

Comment:
This doesn't seem like a reliable method to detect the platform.  Even if our current images have this setup, if it ever changes, it may not occur to someone to fix this function.

How about:
```
source /etc/os-release
echo \${NAME}
```